### DISCUSSION TITLE
GitHub Desktop Cannot open repo's with VS

### DISCUSSION THREAD

**Discussion by masonempey** [community/community#106343] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello I have running into this issue for a while now and cannot find any resources to help fix it. When I am using GitHub Desktop and I am trying to open my code with a text editor the default is set to VS Code but I want VS 2022 the issue is when I go to options and use the drop down menu to select a default editor the only option is VS Code and not VS 2022. Would much appreciate any feed back of things to try and fix it. Thanks!
![VSBug](https://github.com/community/community/assets/149198668/155a72ef-52f8-46b9-83cd-65e6f0c2be1e)

**Comment by DanielWren48** [comment]
Hi @masonempey,

There is no support for visual studio ide.
Take a look at this discussion, https://github.com/desktop/desktop/issues/3148, its been going on for a while now.
